CONTRIBUTING.md

```diff
@@ -2,10 +2,52 @@
 
 Thanks for your interest in Gotify!
 
-When contributing to gotify/\*, please first discuss the change (typos excluded) you wish to make via issue, chat
-or any other method with the owners of this repository. If you want to fix an already existing issue please write 
-so in a comment.
+First of all, please note that we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project. 
 
-Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+If you have any questions you can join the chat on [#gotify:matrix.org](https://matrix.to/#/#gotify:matrix.org).
 
-If you have any questions you can join the chat on [#gotify:matrix.org](https://matrix.to/#/#gotify:matrix.org).
\ No newline at end of file
+## Where to Contribute
+
+- [gotify/server](https://github.com/gotify/server) - server implementation and WebUI code. Written in Golang and React Typescript.
```

Comment:
How about this? I kinda like using code formatting for the languages, then it looks like tags (:.
```
- [gotify/server](https://github.com/gotify/server) - the core of this project contains a web server and a web ui `Go` `TypeScript` `React`
- [gotify/android](https://github.com/gotify/android) - mobile app for managing gotify/server  `Java` `Android`
- [gotify/plugin-template](https://github.com/gotify/plugin-template) - template for plugins using [gotify/plugin-api](https://github.com/gotify/plugin-api) `Go`
- [gotify/cli](https://github.com/gotify/cli) - CLI for pushing messages to gotify/server `Go`
- [gotify/website](https://github.com/gotify/website) - documentation [gotify.net](https://gotify.net/) `Markdown`
```
- [gotify/server](https://github.com/gotify/server) - the core of this project contains a web server and a web ui `Go` `TypeScript` `React`
- [gotify/android](https://github.com/gotify/android) - mobile app for managing gotify/server  `Java` `Android`
- [gotify/plugin-template](https://github.com/gotify/plugin-template) - template for plugins using [gotify/plugin-api](https://github.com/gotify/plugin-api) `Go`
- [gotify/cli](https://github.com/gotify/cli) - CLI for pushing messages to gotify/server `Go`
- [gotify/website](https://github.com/gotify/website) - documentation [gotify.net](https://gotify.net/) `Markdown`

Maybe we should also list gotify/contrib here something like `gotify/contrib - community contribution requests`

Reply:
Okay, I have formatted this section into a table so that it looks more lined-up and clean.